### PR TITLE
[FLIN-81] Add relational database to container

### PR DESCRIPTION
**Pull request by MarcosPetrucci**
## User Story
As a user, I would want my app to be containarized, So that it would be easier to maintain

## Acceptance criteria
- [x] Has a relational database container
- [x] Has a mock SQL

## Describe your changes
- Added a `flink-food.sql` containing a mock database plus toy data
- Added the adequate database schema under doc
- Configured the database container

## Github issue this pull requests addresses
Closes issue #9 
Closes issue #10 

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have updated/created the corresponding documentation

## Observations
An automated script to insert data from the SQL file directly into the container was not requested. I tried to do it, but it wasn't simple. As this is a _cool_ feature, not necessary at all, I am creating this PR.

### CODE REVIEW

**Comment by MarcosPetrucci** [comment]
Request @PolimiGiovanniArriciati review specially about the database schema

File: docker-compose.yml [hunk 1]
```diff
@@ -4,24 +4,24 @@ services:
...
   # Observation: the usage of this image of postgres is for testing purposes
-  # TODO: need to updated to connect to the image regarding FLINKFOOD
-  postgres:
+ 
+  postgres: #We are using this standardized postgres image. No need to reinvent the wheel
     image: debezium/postgres:13
     ports:
       - 5432:5432
     healthcheck:
-      test: "pg_isready -U postgresuser -d shipment_db"
```
Comment (PolimiGiovanniArriciati) [inline docker-compose.yml:13]: The internal health check is really awesome!

File: scripts/insert_sql.sh
Comment (PolimiGiovanniArriciati) [inline scripts/insert_sql.sh:1]: We should remove this file

File: docker-compose.yml [hunk 2]
```diff
@@ -4,24 +4,24 @@ services:
   # Source: https://medium.com/event-driven-utopia/configuring-debezium-to-capture-postgresql-changes-with-docker-compose-224742ca5372
   # Source: https://github.com/debezium/container-images/tree/main/postgres/13
   # Observation: the usage of this image of postgres is for testing purposes
-  # TODO: need to updated to connect to the image regarding FLINKFOOD
-  postgres:
+ 
+  postgres: #We are using this standardized postgres image. No need to reinvent the wheel
     image: debezium/postgres:13
```
Comment (PolimiGiovanniArriciati) [inline docker-compose.yml:9]: Not sure about the version, I would use `postgres:16-alpine`

**Review by PolimiGiovanniArriciati — CHANGES_REQUESTED** [review]

File: docker-compose.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ services:
   # Observation: the usage of this image of postgres is for testing purposes
  
   postgres: #We are using this standardized postgres image. No need to reinvent the wheel
-    image: debezium/postgres:13
+    image: debezium/postgres:16-alpine
```
Comment (MarcosPetrucci) [inline docker-compose.yml:9]: Just to let @VegSja and @febagni we changed the postgreSQL image to a newer one given some compatibility issues with Gio's Mac

**Pull request merged**